fastapi-app/config.py

```diff
@@ -0,0 +1,43 @@
+import os
+from pathlib import Path
+from typing import List, Union
+
+from dotenv import load_dotenv  # pip install python-dotenv
```

Comment:
Freeze this into requirements.txt so that you dont write comments to remember package names :)